### ISSUE TITLE
Add explanatory copy above IAM tables

### ISSUE THREAD

**Issue by david-crespo** [oxidecomputer/console#2047]
Effective role logic is somewhat opaque!

![image](https://uploads.linear.app/b7545512-bb0c-4743-adc4-e5797212a353/89b3f85b-6472-4c55-8352-965c6d0f11d8/640b2951-b462-48c3-b412-385cad732200?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiL2I3NTQ1NTEyLWJiMGMtNDc0My1hZGM0LWU1Nzk3MjEyYTM1My84OWIzZjg1Yi02NDcyLTRjNTUtODM1Mi05NjVjNmQwZjExZDgvNjQwYjI5NTEtYjQ2Mi00OGMzLWI0MTItMzg1Y2FkNzMyMjAwIiwiaWF0IjoxNzEwMjk2MzAzLCJleHAiOjMzMjgwODU2MzAzfQ.QSNSTpwv4EyuXfcsIefmmMUqYCA8GGUBI6Ups7-eUnw)

**Comment by charliepark** [comment]
At some point, we'll want to think of a mechanism beyond color for highlighting the effective role, as (I believe) color alone is insufficient from an accessibility context.

One option: We could add an "Effective role" (or whatever we want to call it) column either before or after the existing role columns.

**Comment by charliepark** [comment]
<img width="1215" alt="Screenshot 2024-03-08 at 9 38 42 AM" src="https://github.com/oxidecomputer/console/assets/22547/81933a07-5b87-473c-81e6-b7474ddba90d">

**Comment by david-crespo** [comment]
We just talked about this, and I think what we want to convey is:

* You can get a role directly on your user or from a group membership
* A user or group can have a role on this resource and/or a parent
* The effective role is the strongest role from any of those sources

**Comment by charliepark** [comment]
We also need to make sure that the group membership permissions are accurately represented as part of the individual user's effective role in console. I'm not sure that that's currently the case.

**Comment by benjaminleonard** [comment]
We need to figure out a designed solution for this – some inline documentation would be helpful but the text directly above the tables does not work for me. Will review with Eugene and come back.

**Comment by charliepark** [comment]
We are moving to show the "effective" role on the IAM tables; do we think we still need explanatory copy for this page? If it's a design pattern that we aren't happy with, I think it might be fine to close this issue.

The soon-to-be-merged UI:
![Screenshot 2024-03-19 at 5 21 29 PM](https://github.com/oxidecomputer/console/assets/22547/7cbd1b15-5acb-476a-b8ef-ecb218f6276e)

**Comment by benjaminleonard** [comment]
I'd say we can close, if there's an issue where we can improve the UX and make the interaction clearer we should probably handle that in a design issue first before upstreaming into the console.
